### PR TITLE
Add device-memory/OWNERS

### PR DESCRIPTION
**Pull request by zcorpan**
cc @spanicker @fmeawad

### CODE REVIEW

**Comment by zcorpan** [comment]
@spanicker @fmeawad are you OK with being listed in the OWNERS file here? It means you’ll get notified any time someone submits a PR with Media Capture tests, or changes to existing tests. And the tests could only be merged to the master branch after review from one of the OWNERS.

**Comment by spanicker** [comment]
@tdresser 
Tim - could we add CSM folks as owners here?

**Comment by tdresser** [comment]
Adding myself and @npm1 as owners SGTM.

**Comment by zcorpan** [comment]
Added.

@spanicker @fmeawad @npm1 please confirm whether you are OK with being listed in OWNERS here.

**Comment by npm1** [comment]
SGTM

File: device-memory/OWNERS [hunk 1]
```diff
@@ -1,2 +1,4 @@
 @spanicker
```
Comment (spanicker) [inline device-memory/OWNERS]: could you remove spanicker and fmeawad? 
The other two are appropriate owners.

**Pull request merged**